scorecard_vanilla.rb

```diff
@@ -0,0 +1,55 @@
+# This is the 'vanilla' scorecard. It doesn't do anything fancy, 
+# it has no classes, it just does what is asked for on the assignment
+
+def game
+    p 'welcome to bowling!'
+    count = 0
+    until count == 9
+        count += 1
+        round
+    end
+    last_round
+    p 'congratulations you bowled a game!'
+end
+
+
+def round
+    this_rounds_score = []
+    p 'please enter your first score'
+    first_bowl = gets.chomp().to_i
+    if first_bowl == 10
+        p 'STRIKE!!!'
+        score(10,0)
+    else 
+        p 'please enter your second score'
+        second_bowl = gets.chomp().to_i
+        score(first_bowl, second_bowl)
+    end
+end
+
+def score(x,y)
+    running_score = []
```

Comment:
does this score method calculate the score for the frame, or the total score for the entire game?